config/initializers/filter_parameter_logging.rb

```diff
@@ -1,4 +1,4 @@
 # Be sure to restart your server when you modify this file.
 
 # Configure sensitive parameters which will be filtered from the log file.
-Rails.application.config.filter_parameters += %i(password auth_token)
+Rails.application.config.filter_parameters += [:password]
```

Comment:
According to knock gem (https://github.com/nsarno/knock/blob/master/lib/knock/authenticable.rb#L11) `token` can be passed through parameter. Should we filter it to here?